var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -288,6 +288,11 @@ class Curl(NMakePackage, AutotoolsPackage):
             # curl queries pkgconfig for openssl compilation flags
             depends_on("pkgconfig", type="build")
 
+    with when("platform=windows"):
+        # Curl should be built either static or shared on Windows to avoid
+        # collisions between import library and static library
+        variant("shared", default=True, description="Build shared curl")
```

Comment:
Is there any particular reason to keep this variant as windows specific? It would seem valuable to be able to specify a static build on Linux or macOS as well.

Reply:
Oh I see the `libs` variant now, I'll just use that.